plugins/decklink/decklink-device-mode.hpp

```diff
@@ -23,4 +23,8 @@ class DeckLinkDeviceMode {
 	const std::string& GetName(void) const;
 
 	void SetMode(IDeckLinkDisplayMode *mode);
+
+	int GetWidth();
+
```

Comment:
You can probably remove this blank line to group `GetWidth()` and `GetHeight()`.